src/main/scala/no/ndla/conceptapi/model/domain/Concept.scala

```diff
@@ -33,7 +33,8 @@ case class Concept(
     tags: Seq[ConceptTags],
     subjectIds: Set[String],
     articleId: Option[Long],
-    status: Status
+    status: Status,
+    visualElement: Seq[VisualElement]
```

Comment:
Skulle vi kunne ha flere visuelle elementer eller kun èn som i Topic Article?

Reply:
En per språk.

Reply:
Det Gunnar sa :smile: 
Dette er det som lagres i databasen, så en pr språk.
Fungerer på samme måte for artikler i draft-api og article-api.